WolvenKit/Views/Wizards/ProjectWizardView.xaml.cs

```diff
@@ -1,11 +1,13 @@
 using System.Reactive.Disposables;
 using ReactiveUI;
 using WolvenKit.ViewModels.Wizards;
+using System.Windows.Forms;
```

Comment:
Could you use the WPF variant of the timer?
This one:
https://docs.microsoft.com/en-us/dotnet/api/system.windows.threading.dispatchertimer?view=windowsdesktop-6.0